charts/nri-k8s-operator/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: A Helm chart to deploy the New Relic Kubernetes Operator.
 name: nri-k8s-operator
 version: 0.1.0
-appVersion: 0.1.0
+appVersion: 1.4.0
```

Comment:
Why that?

Reply:
Shouldn't we start releasing from 0.1.0 following semver?

Reply:
Definitely, this is just the same placeholder for the image we're using right now. Will update it to 0.1.0 when we have the real one.